src/test/java/core/basesyntax/service/ParserServiceTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.ParserServiceImpl;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ParserServiceTest {
+    private static ParserService parserService;
+
+    @Rule
+    public ExpectedException exceptionRuleFileNotFound = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        parserService = new ParserServiceImpl();
+    }
+
+    @Test
+    public void parser_InpuIstNullNotOK() {
```

Comment:
```suggestion
    public void parse_nullInput_NotOK() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

fix all test names